common/src/main/scala/com/gu/zuora/encoding/CustomCodecs.scala

```diff
@@ -0,0 +1,57 @@
+package com.gu.zuora.encoding
+
+import com.gu.helpers.StringExtensions._
+import com.gu.i18n.{Country, Currency}
+import com.gu.zuora.encoding.CapitalizationEncoder._
+import com.gu.zuora.model._
+import io.circe._
+import org.joda.time.LocalDate
+
+object CustomCodecs extends CustomCodecs
+
+trait CustomCodecs {
+  //Request encoders
+  implicit val encodeCurrency: Encoder[Currency] = Encoder.encodeString.contramap[Currency](_.iso)
+  //Encodes a Currency as its iso code eg. {"Currency": "GBP"}
+  implicit val encodePaymentGateway: Encoder[PaymentGateway] = Encoder.encodeString.contramap[PaymentGateway](_.name)
+  implicit val encodeCountryAsAlpha2: Encoder[Country] = Encoder.encodeString.contramap[Country](_.alpha2)
+
+  //Account object encoding - unfortunately the custom field name of the Salesforce contact id starts with a lower case
+  //letter whereas all the other fields start with upper case so we need to set it explicitly
+  val salesforceIdName = "sfContactId__c"
```

Comment:
Since `decapitalizeSfContactId` is private, why not make `salesforceIdName` private too?